src/main/java/com/jungdam/common/dto/ResponseMessage.java

```diff
@@ -8,7 +8,8 @@
     MEMBER_READ_SUCCESS(HttpStatus.OK, "회원조회 성공"),
     MEMBER_LOGIN_SUCCESS(HttpStatus.OK, "로그인 성공"),
     TOKEN_REFRESH_SUCCESS(HttpStatus.OK, "토큰 리프레시 성공"),
-    IMAGE_UPLOAD_SUCCESS(HttpStatus.CREATED, "이미지 업로드 성공");
+    IMAGE_UPLOAD_SUCCESS(HttpStatus.CREATED, "이미지 업로드 성공"),
+    DIARY_CREATE_SUCCESS(HttpStatus.CREATED, "일기 생성 성공");
```

Comment:
```suggestion
    DIARY_CREATE_SUCCESS(HttpStatus.CREATED, "일기 생성 성공"),
    ;
```
이 부분 이렇게 작업하면 PR 할때 뭐가 달라졌는지 알아보기가 편하더라구용 ㅠㅠㅠ 어떻게 생각하나요 다들?

Reply:
아하!! 넵넵!! 좋은 것 같아요!! 저렇게 수정하겠슴돠!!

Reply:
오 너무 좋아요. 자바에서 , ; 이거 너무 불편했었는데